### PR TITLE
fiizbuzztree, readme, whiteboard, tests

### PR DESCRIPTION
**Pull request by shaybrow**
- [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: challenges/lib/src/test/java/challenges/FizzBuzzTreeTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class FizzBuzzTreeTest {
+
+    @Test
+    public void fizzBuzzTest () throws Exception {
```
Comment (Mattpet26) [inline challenges/lib/src/test/java/challenges/FizzBuzzTreeTest.java:15]: Good work with your test

File: challenges/lib/src/main/java/challenges/utilities/FizzBuzzTree.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    public static BinaryTree fizzBuzzTree (BinaryTree input) throws Exception {
+        if (input.root == null) throw new Exception("Oh snap, there's nothing to traverse");
+
+        helper(input.root);
+
+ return input;
+
+    }
+    private static void helper(Node inputNode){
```
Comment (Mattpet26) [inline challenges/lib/src/main/java/challenges/utilities/FizzBuzzTree.java:24]: Great work with your helper method! Looks like you really got a handle on recursion!
